app/services/live.js

```diff
@@ -48,9 +60,56 @@ export default class LiveService extends Service {
     }
   }
 
-  async joinSession(userName, userType) {
+  async createRoom(userName) {
+    try {
+      const response = await fetch(`${ENV.BASE_API_URL}/events`, {
+        method: API_METHOD.POST,
+        headers: {
+          'Content-Type': 'application/json',
+        },
+        credentials: 'include',
+        body: JSON.stringify({
+          name: `live-rds-${Math.random()}`,
+          description: 'The RDS live',
+          region: 'in',
+          userId: userName,
+        }),
+      });
+      const { room_id } = await response.json();
+      return room_id;
+    } catch (error) {
+      console.error(error);
+    }
+  }
+
+  async endRoom(roomId) {
+    try {
+      const response = await fetch(`${ENV.BASE_API_URL}/events/end`, {
+        method: API_METHOD.PATCH,
+        headers: {
+          'Content-Type': 'application/json',
+        },
+        credentials: 'include',
+        body: JSON.stringify({
+          id: roomId,
+          reason: 'Session is over',
+          lock: true,
+        }),
+      });
+      const { message } = await response.json();
+      return message;
+    } catch (error) {
+      console.error(error);
+    }
+  }
+
+  async joinSession(userName, role, room) {
     try {
-      const token = await this.getToken(userName, userType);
+      const roomId =
+        ROLES.host === role ? await this.createRoom(userName) : room;
+      console.log({ roomId }); // For now use it to create link for guest
```

Comment:
Why are we logging roomId here?
Is there a significance for the same

Reply:
The host needs to share the invite link with the guests to join the live share. And we need the `roomId` to join. For now, we are getting `roomId` from the log. But I am fixing it in this issue #500 point 3.